e2e/addons/addons_suite_test.go

```diff
@@ -53,6 +57,8 @@ var _ = BeforeSuite(func() {
 var _ = AfterSuite(func() {
 	_, err := cli.AppDelete()
 	Expect(err).NotTo(HaveOccurred())
+	err = aws.DeleteAllDBClusterSnapshots()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
let's do this best effort and not fail the e2e test in case there is an issue with deleting the cluster snapshots
```suggestion
	_ = aws.DeleteAllDBClusterSnapshots()
```

Reply:
Test run forthcoming